### ISSUE TITLE
[bug] Logo in Mod-Menu not shown

### ISSUE THREAD

**Issue by YouDirk** [YouDirk/youdirk_numeric_io#21]
It's a bug in Minecraft Forge. Could fix it. A Pull Request to the official Forge repository is pending

* [Forge/#5775 [Bug] fml|ModInfo.java: logoFile wrong order of priority](https://github.com/MinecraftForge/MinecraftForge/pull/5775)

```diff
index cc1d4eb32..f92333edf 100644
--- a/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModInfo.java
+++ b/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModInfo.java
@@ -50,6 +50,7 @@ public class ModInfo implements IModInfo
     private final ArtifactVersion version;
     private final String displayName;
     private final String description;
+    private final Optional<String> logoFile;
     private final URL updateJSONURL;
     private final List<IModInfo.ModVersion> dependencies;
     private final Map<String,Object> properties;
@@ -74,6 +75,13 @@ public class ModInfo implements IModInfo
                 map(DefaultArtifactVersion::new).orElse(DEFAULT_VERSION);
         this.displayName = modConfig.<String>getOptional("displayName").orElse(null);
         this.description = modConfig.get("description");
+
+        Optional<String> tmp = modConfig.<String>getOptional("logoFile");
+        if (!tmp.isPresent() && this.owningFile != null) {
+            tmp = this.owningFile.getConfig().getOptional("logoFile");
+        }
+        this.logoFile = tmp;
+
         this.updateJSONURL = modConfig.<String>getOptional("updateJSONURL").map(StringUtils::toURL).orElse(null);
         if (owningFile != null) {
             this.dependencies = owningFile.getConfig().<List<UnmodifiableConfig>>getOptional(Arrays.asList("dependencies", this.modId)).
@@ -140,7 +148,7 @@ public class ModInfo implements IModInfo

     public Optional<String> getLogoFile()
     {
-        return this.owningFile != null ? this.owningFile.getConfig().getOptional("logoFile") : this.modConfig.getOptional("logoFile");
+        return this.logoFile;
     }

     public boolean hasConfigUI()
```

**Comment by YouDirk** [comment]
Pending Pull Request

* [Forge/#5775 [Bug] fml|ModInfo.java: logoFile wrong order of priority](https://github.com/MinecraftForge/MinecraftForge/pull/5775)

**Comment by YouDirk** [comment]
Merged into branch 1.14.x

https://github.com/MinecraftForge/MinecraftForge/commit/8c37d642fef2e11febec4007154ddffee20629fa
